provider/kubernetes/kubernetes.go

```diff
@@ -308,6 +310,50 @@ func (p *Provider) loadIngresses(k8sClient Client) (*types.Configuration, error)
 				}
 			}
 		}
+
+		for _, t := range i.Spec.TLS {
+			tlsSecret, exists, err := k8sClient.GetSecret(i.Namespace, t.SecretName)
+			if err != nil {
+				log.Errorf("Unable to retrive secret %s/%s: %s", i.Namespace, t.SecretName, err)
+				continue
+			}
+			if !exists {
+				log.Warnf("Unable to find secret %s/%s", i.Namespace, t.SecretName)
+				continue
+			}
+			_, tlsCrtExists := tlsSecret.Data["tls.crt"]
+			_, tlsKeyExists := tlsSecret.Data["tls.key"]
+			if !tlsCrtExists || !tlsKeyExists {
+				log.Warnf("Secret %s/%s is not of type 'kubernetes.io/tls'", i.Namespace, t.SecretName)
```

Comment:
Is the error message correct?

Reply:
Sorry, no, this message isn't correct anymore